killswitch.py

```diff
@@ -277,6 +277,25 @@ def check_killswitch(
         log.info('Rules applied successfully, allowing execution to continue')
         return False, new_data
 
+    def check_multiple_killswitches(self, data: UPDATABLE_DATA, *names: str, log=logger, version=_current_version):
```

Comment:
declare return type ?

Reply:
I *think* I hadnt pushed changes or something for this. They should be there now.